### PR TITLE
Bug 1931615: OVS Config: fixes detecting bond NM files with static IP

### PR DESCRIPTION
**Pull request by trozet**
With static IP addressing we try to copy the original NM keyfile so that
we can modify it and retain the original IP settings. In the case where
a bond was configured with a static IP address we grep for the UUID in
the keyfiles to determine which keyfile belongs to the original
connection. However, bond slaves will also contain the UUID in their
keyfile, and the grep was accidentally returning multiple files which
causes the script to fail.

Signed-off-by: Tim Rozet <trozet@redhat.com>
(cherry picked from commit e73bd012f14fe5025ccae35059c02d4835ecf4e8)

### CODE REVIEW

**Comment by trozet** [comment]
@danwinship PTAL

**Review by yuqi-zhang — APPROVED** [review]
Backport of https://github.com/openshift/machine-config-operator/pull/2424, out of curiosity, why not use the bot to cherry pick?

**Comment by trozet** [comment]
> Backport of #2424, out of curiosity, why not use the bot to cherry pick?

Just habit. I'll try to remember to use it in the future.

**Comment by yuqi-zhang** [comment]
/hold
until cherry-pick approval. Should the bot be retesting optional tests?

**Pull request merged**
